### ISSUE TITLE
"Configure Estimator with optimization levels" documentation missing imports

### ISSUE THREAD

**Issue by pandasa123** [Qiskit/qiskit-ibm-runtime#667]
**Describe the bug**

[Configure Estimator with optimization levels](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/how_to/error-suppression.html#example-configure-estimator-with-optimization-levels) page has a bug where the `psi1`, `H1`, and `theta1` objects aren’t defined

**Steps to reproduce**

If a user copies the `Configure Estimator with optimization levels` code and runs locally, they receive an error stating `name 'psi1' is not defined`

**Expected behavior**

This job should be able to run without error and produce `psi1_H1`. 

**Suggested solutions**

It's a pretty quick fix. The following lines should be added

```python
from qiskit.circuit.library import RealAmplitudes
from qiskit.quantum_info import SparsePauliOp

psi1 = RealAmplitudes(num_qubits=2, reps=2)
H1 = SparsePauliOp.from_list([("II", 1), ("IZ", 2), ("XI", 3)])
theta1 = [0, 1, 1, 2, 3, 5]
````

But I'd also recommend removing the `1`s since users only have one `psi`, `H`, `theta` parameter for this example (rather than the [original Estimator overview where I stole the snippet from](https://qiskit.org/documentation/apidoc/primitives.html)). 

Here's the complete alternative example:

```python
from qiskit_ibm_runtime import QiskitRuntimeService, Session, Estimator, Options
from qiskit.circuit.library import RealAmplitudes
from qiskit.quantum_info import SparsePauliOp

service = QiskitRuntimeService()
options = Options(optimization_level=2)

psi = RealAmplitudes(num_qubits=2, reps=2)
H = SparsePauliOp.from_list([("II", 1), ("IZ", 2), ("XI", 3)])
theta = [0, 1, 1, 2, 3, 5]

with Session(service=service, backend="ibmq_qasm_simulator") as session:
    estimator = Estimator(session=session, options=options)
    job = estimator.run(circuits=[psi], observables=[H], parameter_values=[theta])
    psi1_H1 = job.result()
```

**Additional Information**

- **qiskit-ibm-runtime version**:
- **Python version**:
- **Operating system**:

**Comment by beckykd** [comment]
Fixing via this commit: https://github.com/Qiskit/qiskit-ibm-runtime/pull/635/commits/6ac0dcd34168a9eb2b7f29cddc39f3f68291d9ce

FYI @kt474 @pandasa123

**Comment by kt474** [comment]
fixed by #635, closing
